backend/app/environments/router.py

```diff
@@ -1,17 +1,56 @@
-from app.environments.schema import Environment
-from app.environments.service import EnvironmentService
-from fastapi import APIRouter, Depends
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, Path, status
 from sqlalchemy.orm import Session
+
 from app.database.database import get_db_session
+from app.dependencies import only_for_admin
+from app.environments.enums import PlatformTypes
+from app.environments.schema import CreatePlatform, Environment, Platform
+from app.environments.service import EnvironmentService
 
 router = APIRouter(prefix="/envs", tags=["environments"])
 
 
 @router.get("")
 def get_environments(db: Session = Depends(get_db_session)) -> list[Environment]:
-    return EnvironmentService().get_environments(db)
+    return EnvironmentService(db).get_environments()
```

Comment:
is there a performance gain with moving the db?
I'm not sure what the best practice way is in this case. Can you still be sure you will always use the latest session? I see no reason why not but want to double check